packages/components/src/PieChart/PieChartIcon.component.js

```diff
@@ -301,12 +305,13 @@ export function PieChartIconComponent({
 	// to keep only the event listener from the TooltipTrigger.
 	const omitI18N = omit(rest, ['i18n', 'tReady']);
 	return (
-		<span className={classnames(theme['tc-pie-chart-icon'], 'tc-pie-icon-chart')} {...omitI18N}>
+		<span className={classnames(theme['tc-pie-chart-icon'], 'tc-pie-icon-chart')}>
 			<svg
```

Comment:
This seems to be a breaking change, and the commit don't mentioned it

Reply:
Look at the PR description and the dedicated wiki page